### ISSUE TITLE
Make "world" variable

### ISSUE THREAD

**Issue by joshuaadickerson** [mpj/unprecedented-midwife#11]
Maybe you're saying "hello world", maybe you're saying "hello MPJ". The greetee (what is the right word?) should be variable. That should be injected in the DIC. It should default to "world". The greetee should be added to strings.helloWorld using sprintf-js.

**Comment by tiagojdf** [comment]
Yes, but this variable should default not just to world, but to the translation of world, because you don't want to break the translation of hello world for all the international users!

**Comment by ngergo6** [comment]
You should also be able to define a template in your translations then. Maybe there are languages where it's ```[Name] [Greeting]``` and not ```[Greeting] [Name]```.

**Comment by rpunkfu** [comment]
Project's own templating language just for this feature haha

**Comment by joshuaadickerson** [comment]
@ngergo6 that's the point of using sprintf-js ;)
